environments/core.dependencies.txt

```diff
@@ -0,0 +1,11 @@
+python>=3.6
```

Comment:
latest merge should have removed these .txt files (only the yml files are present now)